09_tabular.ipynb

```diff
@@ -7367,7 +7369,7 @@
    "source": [
     "We can create a random forest just like we created a decision tree. Except now, we are also specifying parameters which indicate how many trees should be in the forest, how we should subset the data items (the rows), and how we should subset the fields (the columns).\n",
```

Comment:
the above is a nightly rather than stable. is this deliberate?

Reply:
Yes we need the latest version. It'll be out by the time the book is out.